2-mandatory/6-journey-planner.js

```diff
@@ -92,7 +98,9 @@ function isAccessibleByTransportMode() {}
    - Returns the name of the location
       e.g: "Tower Bridge"
 */
-function getLocationName() {}
+function getLocationName(array) {
```

Comment:
Could you check if you can fetch the value based on the problem statement? Test the code.